README.md

```diff
@@ -74,16 +74,16 @@ After finishing a sprint we hold a retrospective to discuss what went well, what
 
 ##Exchange Program
 
-In order to promote personal growth and inter-team knowledge transfer, the Mobile & TV Group offers an exchange program. A exchange is a single or multi-sprint-long period where a colleague from another group can contribute to Mobile & TV product development in a structured way. This allows the opportunity to learn new platforms, languages, and team processes.
+In order to promote personal growth and inter-team knowledge transfer we offer an engineering exchange program. An exchange is a single or multi-sprint-long period where an engineer from another group can contribute to one of our projects in a structured way. This is an opportunity for exchange participants to learn new platforms, languages, and team processes.
 
```

Comment:
This was moved to ```colleague``` to acknowledge non-engineers can take advantage. Is that no longer the case?